parquet-common/src/main/java/org/apache/parquet/bytes/BytesInput.java

```diff
@@ -75,14 +74,27 @@ public static BytesInput concat(List<BytesInput> inputs) {
   public static BytesInput from(InputStream in, int bytes) {
     return new StreamBytesInput(in, bytes);
   }
-  
+
   /**
-   * @param buffer
-   * @param length number of bytes to read
-   * @return a BytesInput that will read the given bytes from the ByteBuffer
+   * @param buffers
+   * @return a BytesInput that will read the given bytes from the ByteBuffers
    */
-  public static BytesInput from(ByteBuffer buffer, int offset, int length) {
```

Comment:
ByteInput is internal, so I think it is fine.

Reply:
I agree if it is internal but I am always confused which class is for public and which is for internal use only. It would be great if someone who have the requried knowledge (not pointing to you...) would mark these classes.